guides/submit-an-issue.md

```diff
@@ -0,0 +1,30 @@
+# Guidelines for Contribution
+###### Last updated March 15, 2018
+
+:::
+#####  Found a Bug?
+
+Before you [submit an issue](https://github.com/HealthCatalyst/Fabric.Cashmere/issues), search the archive, maybe your question was already answered.
+
+If your issue appears to be a bug, and hasn't been reported, open a new issue. Help us to maximize the effort we can spend fixing issues and adding new features by not reporting duplicate issues. Providing the following information will increase the chances of your issue being dealt with quickly:
+
+- **Overview of the Issue** - if an error is being thrown a non-minified stack trace helps

```

Comment:
It might be nice visually to have the 2nd-line indents match the 1st-line indents in ul > li. I tried but my CSS is not up to par.